fission/environment.go

```diff
@@ -57,20 +57,16 @@ func envCreate(c *cli.Context) error {
 	envExternalNetwork := c.Bool("externalnetwork")
 
 	if len(envBuilderImg) > 0 {
-		envVersion = 2
+		if !c.IsSet("version") {
+			envVersion = 2
+		}
 		if len(envBuildCmd) == 0 {
 			envBuildCmd = "build"
 		}
 	}
 
 	resourceReq := getResourceReq(c)
 
-	// Environment API interface version is not specified and
```

Comment:
Wait why remove this? The new code now just leaves envVersion at zero when neither builder nor version is specified?

Reply:
Oh ok, I will revert this back